packages/edit-post/src/components/sidebar/plugin-document-setting/index.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Defines as extensibility slot for the Settings sidebar
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { createSlotFill, PanelBody } from '@wordpress/components';
+import { compose } from '@wordpress/compose';
+import { withPluginContext } from '@wordpress/plugins';
+
+export const { Fill, Slot } = createSlotFill( 'PluginDocumentSettingPanel' );
+
+const PluginDocumentSettingFill = ( { className, title, children, isOpened, onTogglePanel, ...props } ) => (
+	<Fill>
+		<PanelBody
+			className={ className }
+			title={ title }
+			opened={ isOpened }
```

Comment:
If we decide to replicate the logic with toggling used for other panels, this props shouldn't be exposed but rather leave it as an internal thing. We might offer `initialOpen` (https://github.com/WordPress/gutenberg/blob/4371354d77c8b39c99628c590effd560dbe795a3/packages/components/src/panel/README.md#initialopen) instead but it might be confusing as it would make sense to apply it only until it gets toggled and persisted in the browser's storage. There still might be value to it 🤷‍♂

Reply:
I removed it for now and left it internal.